nvflare/app_common/abstract/generic_task.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Dict, Optional
+
+from nvflare.app_common.abstract.fl_model import FLModel
+
+
+class GenericTask(FLModel):
```

Comment:
I prefer just call this `EmptyTask`? or `ControlTask` or `PingPongTask` or something that is more close to the usage of this.

Reply:
Instead of empty task, shall we have a "CommandTask":
params = {"command": "enrich"}
and at the end of the local run, it expect
FLModel(params={"task_status": "enrich_done"})?